### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by Ashkanthegreat**
Created Games and Games test files.

### CODE REVIEW

File: lib/teams.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Team
+
+  def initialize(params)
+    @params = params
```
Comment (reid-andrew) [inline lib/teams.rb]: @params instance var exists only in this commit. Replaced with individual variables in later commits.

File: test/game_teams_test.rb [hunk 2]
```diff
@@ -14,15 +14,15 @@ def setup
...
-      goals: 2,
-      shots: 8,
-      tackles: 44,
-      pim: 8,
-      powerPlayOpportunities: 3,
-      powerPlayGoals: 0,
-      faceOffWinPercentage: 44.8,
-      giveaways: 17,
-      takeaways: 7
+      goals: '2',
```
Comment (reid-andrew) [inline test/game_teams_test.rb:17]: Refactored to recognize incoming data as strings instead of Ints/Floats

File: lib/game_teams.rb [hunk 3]
```diff
@@ -1,7 +1,37 @@
...
+                  :tackles,
+                  :pim,
+                  :powerPlayOpportunities,
+                  :powerPlayGoals,
+                  :faceOffWinPercentage,
+                  :giveaways,
+                  :takeaways
+                  
   def initialize(params)
-    @params = params
```
Comment (reid-andrew) [inline lib/game_teams.rb]: Removed dummy params variable & replaced with actual parameters

File: test/game_teams_test.rb [hunk 4]
```diff
@@ -10,7 +10,7 @@ def setup
     params = {
       game_id: "2012030221",
       team_id: "3",
-      HoA: "away",
+      hoa: "away",
```
Comment (reid-andrew) [inline test/game_teams_test.rb:13]: Atom was seeing "HoA" as a class due to first letter capitalized. Renamed - need to verify that column conversion to symbol still maps correctly.

File: test/teams_test.rb [hunk 5]
```diff
@@ -14,7 +14,7 @@ def setup
     franchiseId: "23",
     teamName: "Atlanta United",
     abbreviation: "ATL",
-    Stadium: "Mercedes-Benz Stadium",
+    stadium: "Mercedes-Benz Stadium",
```
Comment (reid-andrew) [inline test/teams_test.rb:17]: Atom was seeing "Stadium" as a class due to first letter capitalized. Renamed - need to verify that column conversion to symbol still maps correctly.

**Review by Joshua-Tukman — APPROVED** [review]
Looks great! Now we can move forward with the StatTracker class.

**Pull request merged**